File: src/main/java/com/xxx/misc/test/testServiceImpl.java
package com.xxx.misc.test;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

@Service
public class testServiceImpl implements testService {
    @Autowired
    private testMapper testmapper;

    @Override
    public Test fid(Integer id) {
        return testmapper.fid(id);
    }
}

File: src/main/java/com/xxx/misc/test/testServiceImpl.kt
package com.xxx.misc.test

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service

@Service
class testServiceImpl : testService {
    @Autowired
    private lateinit var testmapper: testMapper


    override fun fid(id: Int): Test? {
        return testmapper.fid(id)
    }
}
